src/attestation/Attestation.ts

```diff
@@ -17,12 +17,39 @@ import TxStatus from '../blockchain/TxStatus'
 import { factory } from '../config/ConfigLog'
 import Identity from '../identity/Identity'
 import IAttestation from '../types/Attestation'
-import IRequestForAttestation from '../types/RequestForAttestation'
 import { revoke, query, store } from './Attestation.chain'
+import ICType from '../types/CType'
+import IPublicIdentity from '../types/PublicIdentity'
+import { IDelegationBaseNode } from '../types/Delegation'
 
 const log = factory.getLogger('Attestation')
 
 export default class Attestation implements IAttestation {
+  public static async query(claimHash: string): Promise<Attestation | null> {
```

Comment:
this has to do with the resolved merging conflict, maud changed the order in 2a5e66128ff59237db3ea591ef9edb54ef20bf08 Mn 157 - doc public methods Attestation (#174).
Once both are on develop this can be resolved, before that, I have to have these in my files for development. Could be resolved rebasing my branch on updated develop